projects/plugins/jetpack/modules/contact-form/js/form-style.js

```diff
@@ -0,0 +1,52 @@
+jQuery( function ( $ ) {
+	$( document ).ready( function () {
+		$( '.contact-form .jetpack-select' ).selectmenu();
+
+		generateStyleVariables( '.wp-block-jetpack-contact-form-container' );
```

Comment:
Because of how the jQuery UI Selector works, there's a problem adding the style variables to the form container.
The selector popover lives outside the form, and it's not getting the styles correctly.
I tried adding the style variables to a higher element in the DOM, but it didn't work well.
Need to investigate it a bit deeper.

Reply:
What's the issue with the selector exactly?

Reply:
I managed to fix the issue.
It was happening because I was adding the CSS variables to the Form container element, but the Popover jQuery UI creates is attached to the end of the ` <body>`, so it wasn't getting styled.
I tried changing the selector to add the styles to the body element, but then I started facing an issue in the Editor.
I managed to fix it by adding a check to verify if it's the frontend page. if so, I add the style variables to the body of the page.